.github/workflows/release-draft.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+  # push: to remove above once test is ove
+  #   branches:
+  #     - develop
+  workflow_dispatch:
+    inputs:
+      FROM_TAG:
+        description: 'previous release tag version'
+        required: true
+        default: '5.0.0'
```

Comment:
Please add a note on the github template for issues of type `release` that this will have to be bumped at some point, and when

Reply:
Do we have to update it? Because these are only used as placeholder whenever someone uses the workflow_dispatch under github action, but if this action is automatically run when it merges to develop, these values are not relevant as they're empty (because it wasn't triggered by the workflow)